### PR TITLE
Logout

### PR DESCRIPTION
**Pull request by JohnThirlaway1**
connects to #26

### CODE REVIEW

File: app/components/Navbar.js [hunk 1]
```diff
@@ -4,33 +4,55 @@ var Router = ReactRouter.Router;
 var Route = ReactRouter.Route;
 var Link = ReactRouter.Link;
 var IndexLink = ReactRouter.IndexLink;
-var NavbarComponent = React.createClass({
 
+var NavbarComponent = React.createClass({
+  getInitialState: function() {
```
Comment (AlexJYoungman) [inline app/components/Navbar.js]: Space after function
`function () {`

Comment (AlexJYoungman) [inline app/components/Navbar.js]: check file for more of these.

File: app/components/Navbar.js [hunk 2]
```diff
@@ -4,33 +4,55 @@ var Router = ReactRouter.Router;
...
+    return { showAltLogin: false };
+   },
+   onClick: function() {
+     this.setState({ showAltLogin: true });
+   },
+   function() { return {query: ''} },
```
Comment (AlexJYoungman) [inline app/components/Navbar.js]: Spaces around items in curly braces
`{ query: '' }`

File: app/components/Navbar.js [hunk 3]
```diff
@@ -6,17 +6,17 @@ var Link = ReactRouter.Link;
 var IndexLink = ReactRouter.IndexLink;
 
 var NavbarComponent = React.createClass({
-  getInitialState: function() {
+  getInitialState: function () {
     return { showAltLogin: false };
    },
```
Comment (TheGarrett) [inline app/components/Navbar.js]: This section is indented 1 space too much

File: app/styles/css/navigation.css [hunk 4]
```diff
@@ -13,7 +13,7 @@
 
 .user {
   display: inline-block;
```
Comment (TheGarrett) [inline app/styles/css/navigation.css]: Theres commented out code in this page, remove it

Comment (TheGarrett) [inline app/styles/css/navigation.css]: Indentation on this page needs sorted

File: app/components/Navbar.js [hunk 5]
```diff
@@ -4,33 +4,55 @@ var Router = ReactRouter.Router;
 var Route = ReactRouter.Route;
 var Link = ReactRouter.Link;
 var IndexLink = ReactRouter.IndexLink;
-var NavbarComponent = React.createClass({
 
+var NavbarComponent = React.createClass({
+  getInitialState: function () {
+    return { showAltLogin: false };
```
Comment (TheGarrett) [inline app/components/Navbar.js]: Indentation on this section is 1 space too much

**Pull request merged**